src/AccessLogEntry.h

```diff
@@ -177,7 +177,7 @@ class AccessLogEntry: public RefCountable
     SBuf lastAclData; ///< string for external_acl_type %DATA format code
 
     HierarchyLogEntry hier;
-    HttpReply *reply = nullptr;
+    HttpReplyPointer reply; ///< virgin 1xx reply or a reply a to-be-sent to client
```

Comment:
Similar to another comment in this review, please use the being-formalized terminology. For example:
```suggestion
    HttpReplyPointer reply; ///< the last sent reply
```